k8s/flink-operator/schema.yaml

```diff
@@ -44,7 +44,7 @@ properties:
     description: >-
       Name of a service account in the target namespace that
       has cluster-admin permissions. This is needed for the operator
-      to create Spark CRD and jobs.
+      to create Flink CRD and jobs.
```

Comment:
Just double-checking, was this supposed to be Flink or should we be leaving it as Spark?

Reply:
Flink it is.